CHANGELOG.md

```diff
@@ -37,14 +49,12 @@ Sooner to that time, we will announce the specific version of TimescaleDB in whi
 **Features**
 * #4874 Allow joins in continuous aggregates
 * #4926 Refactor INSERT into compressed chunks
+* #5212 Allow pushdown of reference table joins
```

Comment:
@svenklemm this was not mentioned on the original PR for 2.10.0 - did we just forget to add back then or should this go to 2.10.1? Or is it a new feature and should go to 2.11.0? (but I also don't see it in the [milestone](https://github.com/timescale/timescaledb/milestone/68) as well)

Reply:
You are right this did not make it into 2.10 and the entry is misplaced.

Reply:
Since this is not a bugfix this should go into 2.11